File: java/program36.java
import java.util.*;

class Number 
{
    public int iNo;

    public Number(int no)
    {
        iNo = no;
    }

    public void Display()
    {
        for(int iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print("(^_^)\t");
        }
    }
}

class program36
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);  
        int iValue = 0;

        System.out.println("Enter number");
        iValue = sobj.nextInt();

        Number obj = new Number(iValue);
        obj.Display();
    }
}

File: cpp/program36.cpp
// program95.c

#include<iostream>
using namespace std;

class Pattern
{
    public:
        int iNo;

    Pattern(int X)
    {
        iNo = X;
    }

    void Display()
    {
        int iCnt = 0; 

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            cout<<"(^_^)"<<"\t";
        }
        cout<<"\n";
    }
};

int main()
{
    int iValue = 0;

    cout<<"Enter Frequency"<<"\n";
    cin>>iValue;

    Pattern pobj(iValue);
    pobj.Display();

    return 0;
}